Sources/MapboxDirections/VisualInstructionComponent.swift

```diff
@@ -177,7 +177,7 @@ extension VisualInstruction.Component: Codable {
         case abbreviatedText = "abbr"
         case abbreviatedTextPriority = "abbr_priority"
         case imageBaseURL
-        case imageURL = "url"
+        case imageURL
```

Comment:
The raw value of this coding key still needs to be `url` for shields, right? Let’s create a separate coding key for that case then.

Reply:
Never mind, the `image` case uses the `imageBaseURL` coding key one line above. This is fine.